src/manifolds/Rotations.jl

```diff
@@ -553,6 +557,10 @@ where tangent vectors are represented by elements from the Lie group
 project_tangent(::Rotations, ::Any...)
 project_tangent!(M::Rotations, w, x, v) = w .= (v .- transpose(v))./2
 
+function represent_in_basis(M::Rotations, x, v, B::ArbitraryOrthonormalBasis) where N
```

Comment:
while this reads ok, the inverse reads really strange. Can we think about another name? `basis_coefficients_of(M,x,v,B)` maybe? for the inverse I only had the idea of `compose` but that name is taken right?

Reply:
Also, docstrings would be nice :)

Reply:
`compose` is taken for groups, right. I don't really have a better ideas for a name.

I'll add more docstrings later.

Reply:
Yes, I also don't have a very good name, but pondering around on the wikipedia page ( https://en.wikipedia.org/wiki/Basis_(linear_algebra) ) maybe we could call

* `represent_in_basis` -> `get_coordinates` (in the sense: get coordinates of `v` with respect to the basis `B`)
* `inverse_represent_in_basis` -> `get_vector` (in the sense: get vector of the coordinates `w` with respect to the basis `B`, though this name sounds a little too generic.

Reply:
OK, `get_coordinates` and `get_vector` sound better. I'll change the code to use these names.